samples/subsys/zbus/remote_mock/remote_mock.py

```diff
@@ -0,0 +1,51 @@
+# Copyright (c) 2022 Rodrigo Peixoto <rodrigopex@gmail.com>
```

Comment:
Can you add a `#!/usr/bin/env python3` and `chmod +x` this?

(same for `uart_bridge/decoder.py`)